docs/platforms/javascript/common/migration/v7-to-v8/v8-new-performance-api.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: New Performance Monitoring APIs
+sidebar_order: 8997
+description: "Learn about new Performance Monitoring APIs in Sentry JavaScript SDK 8.x"
+---
+
+The SDK `8.x` release introduces new APIs for performance monitoring. These APIs are designed to provide more control over how performance data is collected and reported to Sentry.
+
+These APIs are also available in `7.x` so you can incrementally moveover to the new APIs before upgrading to `8.x`.
+
+Previously, there were two key APIs for adding manual performance instrumentation to your applications:
+
+- `startTransaction()`
+- `span.startChild()`
+
+This showed the underlying data model that Sentry was originally based on, which is that there is a root Transaction which can have a nested tree of Spans.
+
+In the new model, transactions are conceptually gone. Instead, you will always operate on spans, no matter where in the tree you are. Note that in the background, spans may still be grouped into a transaction for the Sentry UI. However, this happens transparently, and from an SDK perspective, all you have to think about are spans.
+
+Instead of manually starting & ending transactions and spans, the new model does not differentiate between these two. Instead, you always use the same APIs to start a new span, and it will automatically either create a new Root Span (which is just a regular span, only that it has no parent, and is thus conceptually roughly similar to a transaction) or a Child Span for whatever is the currently active span.
+
+There are three key APIs available to start spans:
+
+- `startSpan()`
+- `startSpanManual()`
+- `startInactiveSpan()`
+
+All three span APIs take `StartSpanOptions` as a first argument, which has the following shape:
+
+```ts
```

Comment:
l/nit (feel free to ignore): hmm I could have sworn this worked better before but `ts`/`js` renders to 

![image](https://github.com/getsentry/sentry-docs/assets/8420481/6e2760a8-780f-4c94-904b-e2f52871fe3f)

Should we change to "JavaScript"/"TypeScript" anyway for the meantime?